src/snowcli/plugins/sql/commands.py

```diff
@@ -36,6 +36,9 @@ def execute_sql(
         "-i",
         help="Read the query from standard input. Use it when piping input to this command.",
     ),
+    show_comments: Optional[bool] = typer.Option(
+        False, "--show-comments", "-s", help="Removes comments from the input query"
```

Comment:
Do we need `-s` alias? It can cause conflicts

Reply:
Removed